event_listener/dfusion_db/models.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
```

Comment:
This annotations library was not found on my end and does not appear to be in use here. What is it for?

Reply:
For evaluating class annotations after class definitions. You'll need python 3.7 for it